test/restart/agent_config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this is a placeholder agent config that common module expects. It is not used to run the agent.